server/eclipse-project/src/main/resources/us/freeandfair/corla/default.properties

```diff
@@ -10,3 +10,8 @@ hibernate.url = jdbc:postgresql://localhost:5432/corla
 hibernate.user = corla
 hibernate.pass = corla
 hibernate.dialect = org.hibernate.dialect.PostgreSQL9Dialect
+hibernate.c3p0.min_size = 5
+hibernate.c3p0.max_size = 20
```

Comment:
I haven’t even checked to see that c3p0 works for connection pooling automatically yet without further configuration, but I had seen 20 in the “how to set this up” docs I read about it. In reality I’d be shocked if we even need a connection pool half that size. But if the default is 100 now it might be worth sticking to that.